aries-site/src/pages/whats-new.mdx

```diff
@@ -53,6 +53,39 @@ either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## January 17-23, 2021
+
+### Background colors
+
+- Incorporated feedback and addressed clarifying questions:
+  - Provided additional guidance what to consider when [selecting a basic or layered approach](https://design-system.hpe.design/foundation/background-colors-guidance#basic-or-layered-approach).
+  - Added an example and guidance for using the [basic layout with cards](https://design-system.hpe.design/foundation/background-colors-guidance#basic-layout-with-cards).
+  - Added examples clarifying how an application header should appear and behave when it is a [fixed header](https://design-system.hpe.design/foundation/background-colors-guidance#with-fixed-header) and another for when the [header scrolls with the content](https://design-system.hpe.design/foundation/background-colors-guidance#header-scrolls-with-content).
+  
+### InfiniteScroll
+
+- Completed [InfiniteScroll design direction](https://www.figma.com/file/XOEwUOqtfLAowprV2psWHM/HPE-Infinite-Scroll?node-id=1%3A35).
+  
+### Layer
+
+- Enhanced ability for layers to have greater control over how [layer corner rounding](https://github.com/grommet/grommet/pull/4888) is applied.
+
+### Pagination
+
+- Incorporated review feedback and incorporated t-shirt sizing. [Play with Pagination behavior](https://deploy-preview-4646--sad-tereshkova-173d07.netlify.app/?path=/story/controls-pagination--custom) 
```

Comment:
Might be worth mentioning that is not completed and still a work in progress? Unless you think it is clear.
For example, something like:
```suggestion
- Incorporated review feedback and incorporated t-shirt sizing. [Play with Pagination POC behavior](https://deploy-preview-4646--sad-tereshkova-173d07.netlify.app/?path=/story/controls-pagination--custom) 
```

Reply:
Good call. Yes, that could be made more clear.

Reply:
Done.